### ISSUE TITLE
"Next event" section should have an explicit `h2` and a uniquely labelled navigation

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#194]
## Description

On the specific event pages, there's an awesome "next event" feature at the bottom. However, since it's starting a new section but _doesn't_ have a heading, screen reader users may misinterpret that this belongs to the prior section "code of conduct" (although that's likely unlikely).

The larger issue here is that there are multiple `nav` landmarks on the page without labelling which creates something like this (using the landmarks extension).

<img width="162" alt="Screen Shot 2022-09-29 at 8 35 25 PM" src="https://user-images.githubusercontent.com/14154792/193165358-3a3e9be8-a506-47e2-b90e-e385aa664218.png">

## Recommendation

By doing the following, we can resolve all the prior issues and set ourselves up for more flexibility in the future:
- Add an explicit `h2` called "More events" (generic enough to cover a future case where we recommend more than one event)
- Use `aria-labelledby` to label the `nav` landmark with the `h2` 

That will end up looking like this:

<img width="1790" alt="Screen Shot 2022-09-29 at 8 32 16 PM" src="https://user-images.githubusercontent.com/14154792/193165476-db5cd51a-e3c9-498a-a45e-c6093efeb202.png">

<img width="160" alt="Screen Shot 2022-09-29 at 8 37 07 PM" src="https://user-images.githubusercontent.com/14154792/193165494-b38a6927-51dc-4af3-924f-c9fbc60f4578.png">
